test/functorch/test_control_flow.py

```diff
@@ -2838,7 +2838,7 @@ def forward(self, fct_1, init_1, xs_1):
     clone = torch.ops.aten.clone.default(init_1);  clone = None
     select_copy = torch.ops.aten.select_copy.int(xs_1, 0, 0);  select_copy = None
     scan_combine_graph_0 = self.scan_combine_graph_0
-    scan = torch.ops.higher_order.scan(scan_combine_graph_0, [init_1], [xs_1], 0, True);  scan_combine_graph_0 = init_1 = xs_1 = None
+    scan = torch.ops.higher_order.scan(scan_combine_graph_0, [init_1], [xs_1], 0, True, []);  scan_combine_graph_0 = init_1 = xs_1 = None
```

Comment:
Btw, I have a slight preference for Tuple over List in general for all three of these lists. The thing about List is that it is mutable and that leads to footguns, if someone writes a scan_dense that mutates the list then we've got a problem.

My question here is: are we being consistent with what we use for the other control flow ops? Let's be consistent with everything. If we're not consistent, then let's pick one of the two to go with

Reply:
It's a bit chaotic right now for cond and scan, it's a list but for while_loop it's a tuple. Let me add a TODO for unifying them.